app/services/publication_state.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class PublicationState
```

Comment:
Maybe move this into Document? it doesn't feel like a service.

Potentially can use: https://til.hashrocket.com/posts/b492fb26ec-rails-enums-and-postgresql-enums ?

Reply:
😞 Doesn't look like custom types work very well with Rails. Normal schema dump won't work with custom types:

<img width="939" alt="screen shot 2018-08-14 at 17 38 59" src="https://user-images.githubusercontent.com/233676/44105241-fc2fb86c-9fe8-11e8-95e6-c3ea74ea3d37.png">

And when I switch to dump as `schema.sql`, I get this error:

```
psql:/var/govuk/content-publisher/db/structure.sql:20: ERROR:  must be owner of extension plpgsql
rake aborted!
failed to execute:
psql -v ON_ERROR_STOP=1 -q -f /var/govuk/content-publisher/db/structure.sql content_publisher_test
``` 

I've moved the constant into `Document` instead.